ios/Sources/Component/View/FeedContentListView.swift

```diff
@@ -0,0 +1,92 @@
+import Model
+import SwiftUI
+
+public struct FeedContentListView: View {
+    private var feedContents: [FeedContent]
+    private let tapContent: (FeedContent) -> Void
+    private let tapFavorite: (_ isFavorited: Bool, _ id: String) -> Void
```

Comment:
This is just a question.
Why is this closure argument type is not `FeedContent` ?

Reply:
looks like repository only need it's id & I just thought there are no need to pass the whole struct.
https://github.com/DroidKaigi/conference-app-2021/blob/17207a9099d23768fdf3150c94d616f91429e308/ios/Sources/Repository/FeedRepository.swift#L38